src/apps/demo-modal/demo-modal.test.ts

```diff
@@ -17,7 +17,10 @@ describe("Opens demo modal", () => {
       .should("contain", "demo-modal-two");
     cy.get(".modal").find("button.modal-btn-close").should("exist");
     // Close the second modal again
-    cy.get(".modal").find("button.modal-btn-close").first().click();
+    cy.get(".modal")
+      .find("button.modal-btn-close")
+      .first()
+      .click({ force: true });
```

Comment:
why force:true?

Reply:
@sinejespersen I'm not sure, actually. That's a part of @kasperbirch1 's changes from PR #200 (My PR won't be merged in before his is approved, so he will answer your question if you ask in #200 ).